cas-server-webapp-support/src/main/java/org/jasig/cas/web/flow/InitialFlowSetupAction.java

```diff
@@ -101,6 +107,11 @@ protected Event doExecute(final RequestContext context) throws Exception {
                         registeredService.getId());
                 WebUtils.putRegisteredService(context, registeredService);
             }
+        } else if (!this.enableFlowOnAbsentServiceRequest) {
```

Comment:
If the service is found, we are ok. But if not, and this option is disabled, the flow would throw an exception informing the user which is the intent of this PR.